javascript/ql/lib/semmle/javascript/security/dataflow/TaintedPathFromLibraryInputCustomizations.qll

```diff
@@ -0,0 +1,8 @@
+import TaintedPathCustomizations
+
+/**
+ * An input to a library is considered a flow source.
+ */
+class LibInputAsSource extends TaintedPath::Source {
+  LibInputAsSource() { this = getALibraryInputParameter() }
```

Comment:
You forgot to import `semmle.javascript.PackageExports`.  
(That is what QL-for-QL is complaining about).

Reply:
Thanks, it's fixed locally!